Makefile

```diff
@@ -47,7 +47,7 @@ install: _output/bin/$(LOCAL_OS)/bootkube
 _output/bin/%: GOOS=$(word 1, $(subst /, ,$*))
 _output/bin/%: GOARCH=$(word 2, $(subst /, ,$*))
 _output/bin/%: GOARCH:=amd64  # default to amd64 to support release scripts
-_output/bin/%: $(GOFILES)
+_output/bin/%: $(GOFILES) $(VENDOR_GOFILES)
```

Comment:
Instead of detecting we can probably just drop the dependencies. `go build` already caches everything and detects what needs to be recompiled.

Reply:
Feel free to leave it as is, but I may do a follow up to remove make dependencies.